lib/babel-config.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+var path = require('path');
+var chalk = require('chalk');
+var figures = require('figures');
+var convertSourceMap = require('convert-source-map');
+var objectAssign = require('object-assign');
+var colors = require('./colors');
+
+function validate(conf) {
+	if (conf === undefined || conf === null) {
+		conf = 'default';
+	}
+
+	// check for valid babel config shortcuts (can be either "default" or "inherit")
+	var isValidShortcut = ['default', 'inherit'].indexOf(conf) !== -1;
```

Comment:
This always seemed weird to me… but now that the variable is shorter why not `conf === 'default' || conf ===' inherit'`?